components/brave_extension/extension/brave_extension/braveShieldsPanel.tsx

```diff
@@ -19,9 +20,17 @@ const store: any = new Store({
 
 Promise.all([
   store.ready(),
-  new Promise(resolve => chrome.braveTheme.getBraveThemeType(resolve))
+  new Promise(resolve => chrome.braveTheme.getBraveThemeType(resolve)),
+  getViewPreferences()
 ])
-.then(([ , themeType ]: [ undefined, chrome.braveTheme.ThemeType ]) => {
+.then((
+  [ , themeType, settings ]:
+  [
+    undefined,
```

Comment:
I know this is just reformatting, but this seems like it should have a comment? Why `undefined` here?

Reply:
@bridiver i didn’t add this, but it’s the `.then(` resolve value of `Promise.all([])` to combine each Promise’s resolve value in an array in the same order that the Promise array is. So this is saying that their either is no resolve value, or we don’t care about the resolve value, of the first Promise: `store.ready()`.

There might be a nicer way to handle Promise.all in typescript so that it provides this typing automatically.